cypress.json

```diff
@@ -21,6 +21,7 @@
     "DATASOURCE_MANAGEMENT_ENABLED": false,
     "ML_COMMONS_DASHBOARDS_ENABLED": true,
     "WAIT_FOR_LOADER_BUFFER_MS": 0,
-    "DASHBOARDS_ASSISTANT_ENABLED": false
+    "DASHBOARDS_ASSISTANT_ENABLED": false,
+    "DASHBOARDS_WORKSPACE_ENABLED": false
```

Comment:
Shall we need start with `DASHBOARDS`? I think workspace is not direct related dashboard.

Reply:
Sure, I will delete the prefix `DASHBOARDS `.